test/default-split-pane-tests.js

```diff
@@ -361,60 +695,80 @@ describe('Drag resizer', () => {
     it('first resizer', () => {
 
       const jsx = (
-        <SplitPane split="horizontal">
+        <SplitPane split='horizontal'>
           <Pane>one</Pane>
           <Pane>two</Pane>
           <Pane>three</Pane>
           <Pane>four</Pane>
         </SplitPane>
       );
 
-      asserter(jsx, {height: 403})
-        .assertRatios([25, 25, 25, 25])
-        .assertSizes([100, 100, 100, 100])
+      asserter(jsx, {height: 400})
```

Comment:
How wide is the resizer you are using? I use `403` as the width to accomodate 3 Resizers of 1 px each

Reply:
Resizer size is 11px, but it has a negative margin of right and left. I've changed it from 5px to -5.5px. So actually resizer does not affect the size of the splitPane container as it overflows part of the pane (in this case by 5.5px) which is not the best solution and I working on it now. The Desirable behavior is if I have an initial size of pane 100px it should be 100px if I have 50% size it should take a percentage of SplitPane container except for resizers.